src/metrics.ts

```diff
@@ -69,6 +69,8 @@ export enum MetricsNames {
   SERVER_CLOSING = 'apollo_server_closing',
   QUERY_STARTED = 'apollo_query_started',
   QUERY_FAILED = 'apollo_query_failed',
+  QUERY_CLIENT_FAILED = 'apollo_client_query_failed',
```

Comment:
it should be `apollo_query_failed_by_client_input`

Reply:
I updated